search-endpoint.md

```diff
@@ -264,20 +282,44 @@ The list of returned facet fields will depend on the Discovery configuration: ht
 The JSON response document is as follow
 ```json
 {
-  "facets": [
-    {
-      "name" : "author",
-      "type" : "string"
-    },
-    {
-      "name" : "dateissued",
-      "type" : "date"
-    },
-    {
-      "name" : "subject",
-      "type" : "string"
+  "scope": null,
+  "configurationName": null,
```

Comment:
we also need the query to be consistent with the objects endpoint (and we need to add the configurationName to the objects endpoit as well)